triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -563,6 +575,10 @@ object Runner extends StrictLogging {
     * commands that may be pending.
     */
   val maxParallelSubmissionsPerTrigger = 8
+  val maxTriesWhenOverloaded = 6
+
+  private def overloadedRetryDelay(afterTries: Int): FiniteDuration =
```

Comment:
Any particular reason for the choice of 250ms? Seems totally sensible but so does 100ms or whatever else you can come up with.

Reply:
Only that it quickly converges to whole seconds.